src/SingleBookingPage/bookingItem/Header.js

```diff
@@ -43,29 +43,31 @@ const Header = (props: Props) => {
   return (
     <div>
       <div className="headerAbove">
-        <div>
+        <div data-cy="booking-type">
           {booking.databaseId && (
             <Text type="secondary">
-              {isFuture ? 'Upcoming' : 'Last'} trip #&nbsp;
```

Comment:
Im not sure if this should really be `Last`?

Reply:
cc @mjancik Last vs Past?

Reply:
this zeplip has last: https://zpl.io/V0E5kgm but on allbookings page it's past